Help/new.html

```diff
@@ -326,4 +326,4 @@
    </div>
   </div>
  </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
Please remove commits that change the HTML. You should only change the subtemplate.